modules/ROOT/pages/clustering/databases.adoc

```diff
@@ -574,12 +576,40 @@ CREATE DATABASE foo OPTIONS { existingData: 'use', seedURI: 'azb://myStorageAcco
 ======
 =====
 
+The `CloudSeedProvider` supports seeding up to a specific date or transaction ID using the `seedRestoreUntil` option.
+
+==== Seed up to a specific date
+
+To seed up to a specific date, you need to pass the differential backup, which contains the data up to that date.
+
+
+[source,shell]
+----
+CREATE DATABASE foo OPTIONS { existingData: 'use', seedURI: 's3:/myBucket/myBackup.backup', seedRestoreUntil: datetime("2019-06-01T18:40:32.142+0100") }
+----
+
+This will seed the database with transactions committed before the provided timestamp.
+
+==== Seed up to a specific transaction ID
+
+To seed up to a specific transaction ID, you need to pass the differential backup that contains the data up to that transaction ID.
+
+[source,shell]
+----
+CREATE DATABASE foo OPTIONS { existingData: 'use', seedURI: 's3:/myBucket/myBackup.backup', seedRestoreUntil: 123 }
+----
+
+This will seed the database with transactions up to, but not including transaction 123.
+
+[role=label--deprecated]
 [[s3-seed-provider]]
 ==== S3SeedProvider
 
+// When Cypher 25 is released, we have to label this section 'Cypher 5' as this functionality is only available in Cypher 5.
+
 The `S3SeedProvider` supports:
 
-** `s3:` label:deprecated[Deprecated in 5.26]
```

Comment:
I think this should still be marked as deprecated. Use of the `S3SeedProvider` _is_ deprecated, we want everyone to use `CloudSeedProvider`, but if they are already using the `S3SeedProvider`, and want to stick with it to be able to pass in different credentials, the docs need to allow them to with the deprecation warning, until Cypher 5 goes away.